packages/signers/src/index.ts

```diff
@@ -24,3 +24,7 @@ export {
   Web3AuthSigner,
   type Web3AuthAuthenticationParams,
 } from "./web3auth/index.js";
+export {
+  ArcanaAuthSigner,
+  type ArcanaAuthAuthenticationParams,
+} from "./arcana-auth/index.js";
```

Comment:
one thing we're trying is tree-shaking the exports by having signers export to sub-paths of the SDK. See [this](https://github.com/alchemyplatform/aa-sdk/blob/ce7deed9/packages/signers/package.json#L29) for reference.

So for arcana, a dev would have to do `export { ArcanaAuthSigner } from @alchemy/signers/arcana`.

Ask: Can you update package.json to export your Signer and types to the "/arcana" subpath of the SDK?

Reply:
I tried to copy like it is in the example you provided, can you confirm whether it works?

Reply:
lgtm